packages/core-users/db/helpers.js

```diff
@@ -219,10 +207,9 @@ Users.enrollUser = async ({ password, email, displayName, address }) => {
     }
   );
   if (!params.password) {
+    const userEmail = Users.findOne({ _id: newUserId }).primaryEmail()?.address;
     // send an e-mail if password is not set allowing the user to set it
-    accountsPassword.sendEnrollmentEmail(
-      Users.findOne({ _id: newUserId }).primaryEmail()?.address
-    );
+    accountsPassword.sendEnrollmentEmail(userEmail);
```

Comment:
This can also be removed

Reply:
The email-verification is the one triggered by default but enrollment email isn't so we need to keep it.

Reply:
Yeah exactly that's why I want to remap the verify-email to enrollAccount action in prepareEmail ;)

Reply:
-> No we can remove it here